fluss-rpc/src/main/proto/FlussApi.proto

```diff
@@ -449,25 +449,27 @@ message NotifyRemoteLogOffsetsRequest {
 message NotifyRemoteLogOffsetsResponse {
 }
 
-message CommitLakeTableSnapshotRequest {
-  repeated PbLakeTableSnapshotInfo tables_req = 1;
+message PrepareCommitLakeTableSnapshotRequest {
```

Comment:
`PrepareLakeTableSnapshotRequest`

`prepare` and `commit` are 2 different phases.